README.md

```diff
@@ -9,29 +9,30 @@ This is my research and reflection journal for DGL 104 winter 2023
 - want to create some new things with learned technology or skill.
 - want to use this in an organization i am going to work and get better in using those new technology.
 - perfect in what i am learning in this course.
-- 
 
 
 ## interests
 
 - *Get* a nice job and make satisfied with my job.
 - up to date with the new technologies.
-- I want to learn Java language, HTML and CSS scripting languages for my  future goal, where these are source to make a website and run's in a effective manner.
+- I want to learn Java language, HTML and CSS scripting languages for my future goal, where these are source to make a website and run's in a effective manner.
 - want to develop one site for my village and local area about things in the places.
```

Comment:
This is a much more specific goal: You want to use HTML, CSS and scripting languages to make better websites. This is more like what I'm looking for when I ask for goals. 

You are still missing a measure. How will you know that you've gotten better? One approach would be to say that you are going to practice coding each day (even for 15 minutes), then you can look back at the end of each week and decide if you were successful or not: if you practiced five days out of seven, that's pretty good! If you only practiced two out seven, maybe that's not so good and you haven't been successful.

Also, Java, unfortunately, isn't going to be very helpful in building websites. However, if you use Java as your language and you get better at Java, that will still help you with your HTML, CSS and especially JavaScript knowledge and skills.